### ISSUE TITLE
Allow projects to specify CHANGELOG location

### ISSUE THREAD

**Issue by ralphbean** [fedora-infra/anitya#237]
It would be cool if we could have some kind of link or specification of where the changelog lives for each project.

Opportunities:

- We could include some kind of pointer to the changelog in the fedmsg payload.
- We could then have the-new-hotness daemon try to extract the changelog snippet for the latest release and add it to the bugzilla ticket.
- FMN notifications about new upstream releases could include the changelog snippet in the ``long_form`` representation.
- This would help packagers more quickly understand if the latest release if something that should be looked at more closely (or if any additional steps should be taken).

Maybe we could use ``pandoc`` for parsing at an additional step down the pipeline.

Problems:

- Some projects don't have a changelog at all, so we need to handle that.
- Some projects keep their changelog as HTML on their site, but it is not included in their tarball.
- Some projects keep their changelog in their source control repo, but it is not included in their tarball.
- Some projects keep it in their tarball.

I don't immediately see a way to allow anitya maintainers to easily specify the location, accounting for all those possibilities.  Will have to think about how best to add this to the db model.

/cc @vdanen

**Comment by razzeee** [comment]
This would also be something https://github.com/flathub/flatpak-external-data-checker would like to consume

**Comment by Zlopez** [comment]
What needs to be done:
* Add new field to project page that allows you to add link to changelog
* Add new filed to [Anitya message schema](https://github.com/fedora-infra/anitya-messages)
* Update the-new-hotness to work with this new message field

For now I would ignore any project that doesn't have changelog on URL.

**Comment by razzeee** [comment]
It would be great, if it could not only consume changelog files, but also github(or else) releases

**Comment by Zlopez** [comment]
@razzeee In this case, it will be just enough to add link to Github releases page.

**Comment by razzeee** [comment]
Yes, but I would also expect anitya to then filter these down and add the correct release notes to the correct release. Did I get that right?

**Comment by Zlopez** [comment]
Maybe this could be an improvement, but in the first iteration just adding changelog link should be enough.
